elastic/kibana
# [Ingest Pipeline] Allow to provide a redirect path and a tab in component template edit UI

## Description 

Related to #133740 

Similar to https://github.com/elastic/kibana/pull/134776 

We need to be able to link the component template edit UI from Fleet on a specific tab and to be redirect to Fleet when the user save.

For this I introduced two query params `?tab=` and `?redirect_path` to select a specific tab and to redirect on save/close.


## How to test

While running Kibana without basepath
Go to `/app/management/data/index_management/edit_component_template/.alerts-observability.logs.alerts-mappings?tab=mappings&redirect_path=/app/fleet` 

You should go directly to the mappings tab and on save be redirected to Fleet 

<img width="1248" alt="Screen Shot 2022-06-22 at 9 43 39 AM" src="https://user-images.githubusercontent.com/1336873/175044084-e32c8422-dddd-43de-a53d-6eadeb523561.png">

This should work from the view component template to `/app/management/data/index_management/component_templates/.alerts-ecs-mappings?redirect_path=/app/fleet`
